src/components/v-menu/v-menu.vue

```diff
@@ -0,0 +1,737 @@
+			hasWindow.value = typeof window !== 'undefined';
+		});
+		onMounted(() => {
+			isActive.value && callActivate();
+		});
+
+		/**
+		 *
+		 * (De)Activate
+		 *
+		 **/
+
+		function getActivator(e?: Event): HTMLElement | null {
+			// If we've already fetched the activator, re-use
+			if (activatorElement.value) return activatorElement.value;
+
+			if (e) {
+				// Activated by a click event
+				const activator = (e.currentTarget || e.target) as HTMLElement;
+				activatorElement.value = activator;
+			}
+
+			return activatorElement.value;
+		}
+		function checkActivatorFixed() {
+			if (attach.value !== false) return; // this.attach !== false
+			let el = getActivator();
+			while (el) {
+				if (window.getComputedStyle(el).position === 'fixed') {
+					activatorFixed.value = true;
+					return;
+				}
+				el = el.offsetParent as HTMLElement;
+			}
+			activatorFixed.value = false;
+		}
+		function activate() {
+			// Update coordinates and dimensions of menu
+			// and its activator
+
+			updateDimensions();
+			// Start the transition
+			requestAnimationFrame(() => {
+				// Once transitioning, calculate scroll and top position
+				startTransition().then(() => {
+					if (content.value !== null) {
+						calculatedTopAuto.value = calcTopAuto();
+						props.auto && (content.value.scrollTop = calcScrollPosition());
+					}
+				});
+			});
+		}
+		function callActivate() {
+			if (!hasWindow) return;
+			activate();
+		}
+		function callDeactivate() {
+			isContentActive.value = false;
+		}
+
+		/**
+		 *
+		 * Calc Position
+		 *
+		 **/
+		const calculatedLeft = computed((): string => {
+			const menuWidth = Math.max(
+				dimensions.content.width,
+				parseFloat(calculatedMinWidth.value)
+			);
+
+			if (!props.auto) return calcLeft(menuWidth) || '0';
+
+			return `${calcXOverflow(calcLeftAuto(), menuWidth) || '0'}px`;
+		});
+		const computedLeft = computed((): number => {
+			const a = dimensions.activator;
+			const c = dimensions.content;
+			const activatorLeft = (attach.value !== false ? a.offsetLeft : a.left) || 0; // this.attach !== false
+			const minWidth = Math.max(a.width, c.width);
+			let left = 0;
+			left += props.left ? activatorLeft - (minWidth - a.width) : activatorLeft;
+			if (props.offsetX) {
+				const maxWidth = a.width;
+				left += props.left ? -maxWidth : a.width;
+			}
+			return left;
+		});
+		function calcLeft(menuWidth: number): string {
+			return `${
+				attach.value !== false // this.attach !== false
+					? computedLeft.value
+					: calcXOverflow(computedLeft.value, menuWidth)
+			}px`;
+		}
+		function calcLeftAuto(): number {
+			return dimensions.activator.left - defaultOffset.value * 2;
+		}
+
+		const calculatedTop = computed((): string => {
+			const top = !props.auto ? calcTop() : `${calcYOverflow(calculatedTopAuto.value)}px`;
+			return top || '0';
+		});
+		const computedTop = computed((): number => {
+			const a = dimensions.activator;
+			const c = dimensions.content;
```

Comment:
Prefer explicit names for vars (you can't find this by CMD+F `a`)

```suggestion
			const activator = dimensions.activator;
			const content = dimensions.content;
```

Reply:
(suggestion just as an example, needs to be changed elsewhere in this function too obviously)